content/DQKD/_index.md

```diff
@@ -0,0 +1,67 @@
+1. Alice wants to generate symmetric keys with Bob, without sending the key material directly between her and Bob
+
+2. Using her cert from Qrypt, and the ID of Bob's Key Management Entity (KME), Alice calls the API for a single key.
+
+2. A moment later, she recieves the key and respective key ID from her KME on site.
+
+3. Behind the scenes, Alice's KME securely transmits the data required for Bob's KME to generate the exact same key through BLAST. 
+
+4. Then, she comfortably shares the key ID over a public channel with Bob, say over Discord or Whatsapp. Bob recieves the key ID.
+
+5. Bob then sends a request to his KME through the same API to replicate the key. 
+
+6. From his on-site KME, Bob then recieves the key material that was generated by Alice, and can then use that material to send sensitive data to Alice.
+See graphic below for a visual. 
+
+![FullPage](images/dqkddiagram.svg)
+
+Alice and Bob have successfully established a post-quantum cryptographic connection without digging 100's of miles of cable or launching multiple satalites. 
+### Technical Implementation Details
+Alice requests a fresh quantum-random key, tagging Bob's server to allow dqkd to make the key available on that server. 
+
+    curl --cert My_Cert.p12 \
+    "https://dqkd-eastus-1.qrypt.net/api/v1/keys/(Bob's KME ID)/enc_keys"
+
+The API defaults to one key of size 256, because Alice only needs one default-size key, she leaves the data fields blank, and recieves a key:
+
+
+    {"keys": [{"key": “actual key”,"key_ID": “key id of actual key”}]}  
+
+{{% notice note %}}
+To request more than the default, use the following format, by adding the following to the data field of the request:
+{{% /notice %}}
+```c
+    curl --cert My_Cert.p12 \
+    "https://dqkd-eastus-1.qrypt.net/api/v1/keys/(Bob's KME ID)/enc_keys" \
+    -d '{"number": 1,  "size":256}'
+```
+
+Alice then sends the key ID to Bob, who already has Alice's KME ID, then sends the request for the Key Alice generated on her end. 
+
+    curl --cert My_Cert.p12 \
+    "https://dqkd-westus-1.qrypt.net/api/v1/keys/(Alice's KME ID)/dec_keys" \
+    -H "Content-Type: application/json" \
+    -d '{"key_IDs": [{"key_ID": “key id of actual key”}]}'
+
+Bob then recieves the same message generated by Alice
+
+    {"keys": [{"key": “actual key”,"key_ID": “key id of actual key”}]}  
+
+From here the key can be used for any number of use cases, with the assurance that the key data is secure. This example is only to show how the system works, there can be any number of Alices or Bobs under a single KME, and unlimited KMEs that communicate with one another in a network. There is also the option to have a Time To Live (TTL) which can time-gate key decodes for shorter than the default of one hour. 
+
+{{% notice note %}}
+We have implemented Qrypt DQKD in accordance with the [ETSI GS QKD 014](https://www.etsi.org/deliver/etsi_gs/QKD/001_099/014/01.01.01_60/gs_qkd014v010101p.pdf) specification.
+{{% /notice %}}
```

Comment:
There is too much assumed knowledge.

Also, some items in the ordered list are context setting, and don't really belong in an ordered list.

Please replace the content with the following:

## What is DQKD?

Alice is at one location (Site A), and Bob is at a different location (Site B). Each location has its own Key Management Entity (KME) that stores keys and makes them available for use by local clients.

Alice wants to generate a symmetric encryption key and store it in the KME at Site A. She also wants to share that key with Bob and store it in the KME at Site B. It should then be possible to retrieve the key from either site, using the same key id. 

Traditional QKD systems use isolated fiber-optic or satellite based networks to send the key material between the sites.

Qrypt's DQKD solution uses Qrypt's [BLAST](https://www.qrypt.com/wp-content/uploads/2022/03/Whitepaper-EverlastingSecurity.pdf#page=9) protocol instead. The advantage of this protocol is that it makes it possible to exchange keys between sites securely over the public internet, without having to establish an isolated fiber-optic or satellite network.

## How it works

A DQKD cluster consists of multiple sites, each of which has a KME that is identified by a unique KME ID.

Each site has a server exposes the same API, which is an instance of the [ETSI GS QKD 014](https://www.etsi.org/deliver/etsi_gs/QKD/001_099/014/01.01.01_60/gs_qkd014v010101p.pdf) specification. 

The cluster is secured using a single certificate trust chain. All endpoints on the cluster enforce mutual TLS, so all clients will need a client cert that is part of the same trust chain to call the endpoints on the respective sites.

### Generating a key - example data flow

1. Alice calls the `enc_keys` endpoint on Site A to generate a key that she wants to share with Bob:

```bash
 curl --cert My_Certs.p12 \
    "https://dqkd-eastus-1.qrypt.net/api/v1/keys/(Bob's KME ID)/enc_keys"
```
2. The DQKD server uses the BLAST protocol to generate a key and a new random key ID, and stores the key in the local KME
3. The key ID and metadata required to re-genrate that key is sent to site B.
4. The DQKD server at site B receives the metadata and uses it to re-generate the key.
5. The key is then stored and associated with the key ID in site B's KME.
6. Alice receives the key material along with the key ID in the response to her original request:

```json
{"keys": [{"key": "KEY_MATERIAL","key_ID": "SOME_KEY_ID"}]}
```

7. Alice sends Bob the key ID.
8. Bob calls the `dec_keys` endpoint on the DQKD server at site B to get a copy of the key:

```bash
 curl --cert My_Cert.p12 \
    "https://dqkd-westus-1.qrypt.net/api/v1/keys/(Alice's KME ID)/dec_keys" \
    -H "Content-Type: application/json" \
    -d '{"key_IDs": [{"key_ID": "SOME_KEY_ID"}]}'
```

9. Bob receives the key material in the response:

```json
{"keys": [{"key": "KEY_MATERIAL", "key_ID": "SOME_KEY_ID"}]} 
```

### Data flow - Illustrated
![FullPage](images/dqkddiagram.svg)